eva_submission/eload_preparation.py

```diff
@@ -181,22 +181,22 @@ def find_genome(self):
                     self.warning(f'Assembly report was not set for {assembly_accession}')
                 self.eload_cfg.set('submission', 'analyses', analysis_alias, 'assembly_fasta', value=assembly_fasta_path)
 
-            contig_alias_put_db(contig_alias_payload, contig_alias_url, contig_alias_user,
+            contig_alias_put_db(self, contig_alias_payload, contig_alias_url, contig_alias_user,
                                 contig_alias_pass)
 
         else:
             self.error('No scientific name specified')
 
 
 @retry(tries=4, delay=2, backoff=1.2, jitter=(1, 3))
-def contig_alias_put_db(contig_alias_payload, contig_alias_url, contig_alias_user, contig_alias_pass):
-    for assembly in contig_alias_payload:
-        response_get = requests.get(os.path.join(contig_alias_url, assembly),
-                                    auth=(contig_alias_user, contig_alias_pass))
-
-        response_get_data = response_get.json()
+def contig_alias_put_db(self, contig_alias_payload, contig_alias_url, contig_alias_user, contig_alias_pass):
```

Comment:
If this is meant to receive `self` to access the logging capabilities then it should be bart of the `EloadPreparation` class and be indented.

Reply:
updated